docs/data-cookbooks/sns.md

```diff
@@ -6,81 +6,17 @@ hide_title: true
 
 # SNS Notification in Workflows
 
-On deployment, an sftracker (Step function tracker) [SNS](https://aws.amazon.com/sns) topic is created and used for messages related to the workflow.
+On deployment, three [SNS topics](https://aws.amazon.com/sns) are created and used for handling notification messages related to the workflow:
 
-Workflows can be configured to send SNS messages containing the Cumulus message throughout the workflow by using the [SF-SNS-Report lambda function](https://www.npmjs.com/package/@cumulus/sf-sns-report).
+- `reportExecutions` - Receives 1 message per workflow execution
```

Comment:
You're right. It would be 2 messages per workflow execution, one for starting and one for completed/failed. I'll update. And yeah, I can add a diagram if you want

Reply:
Does e8aabaa  address it?

Reply:
Ah yes I like that. Makes a lot more sense.